### ISSUE TITLE
What is "hq"?

### ISSUE THREAD

**Issue by evanp** [Open-Earth-Foundation/OpenClimate-Schema#10]
What is the `hq` column in `Actor` and how do we use it? Is it specific for private sector actors?

**Comment by yeozy95** [comment]
Yes, it is specific for private sector actors to denote HQ country of said actor
